code/modules/spells/spell_types/godhand.dm

```diff
@@ -109,3 +109,54 @@
 	M.Stun(40)
 	M.petrify()
 	return ..()
+
+
+/obj/item/melee/touch_attack/duffelbag
+	name = "\improper burdening touch"
+	desc = "Where is the bar from here?"
+	catchphrase = "BA'R A'RP!!"
+	on_use_sound = 'sound/magic/disintegrate.ogg'
+	icon_state = "disintegrate"
+	inhand_icon_state = "disintegrate"
+
+/obj/item/melee/touch_attack/duffelbag/afterattack(atom/target, mob/living/carbon/user, proximity)
+	if(!proximity || target == user || !isliving(target) || !iscarbon(user)) //Roleplay involving touching is equally as bad
+		return
+	if(!(user.mobility_flags & MOBILITY_USE))
+		to_chat(user, "<span class='warning'>You can't reach out!</span>")
+		return
+	if(!user.can_speak_vocal())
+		to_chat(user, "<span class='warning'>You can't get the words out!</span>")
+		return
+	var/mob/living/carbon/M = target
+	var/elaborate_backstory = pick("spacewar origin story", "military background", "corporate connections", "life in the colonies", "anti-government activities", "upbringing on the space farm", "fond memories with your buddy Keith")
```

Comment:
Seems sad to only use this meme RP text on a failed antimagic cast. Can you integrate it into a successful cast too?

Reply:
Sure, the victim can get a message about that.